### PR TITLE
adding an example for setPopup

### PR DESCRIPTION
**Pull request by dasulit**
This adds an example for setting a popup on a marker.

cc/ @mollymerp

### CODE REVIEW

**Comment by mollymerp** [comment]
Thanks so much @dasulit ! This looks great, but it might be cool to use a little image / icon for the `Marker` instead of just a `div` to showcase the added functionality that `Marker` was built for.

Feel free to make your own, or I found these on some public domain icon site
[source](http://www.flaticon.com/free-icons/washington-monument_198816)
image

**Comment by dasulit** [comment]
Good suggestion @mollymerp, I made the marker an image to make this example less abstract!

File: docs/_posts/examples/3400-01-27-set-popup.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---
+<style>
+
+#marker {
+    background-image: url('{{site.baseurl}}/assets/washington-monument.jpg');
+    background-size: cover;
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    cursor: pointer; }
```
Comment (mollymerp) [inline docs/_posts/examples/3400-01-27-set-popup.html]: nitpicky detail -- can you put the closing bracket on a new line for these 2 css rules?

**Comment by dasulit** [comment]
cool, CSS formatting updated

File: docs/_posts/examples/3400-01-27-set-popup.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+var el = document.createElement('div');
+el.id = 'marker';
+
+// create the marker
+new mapboxgl.Marker(el, {offset:[-25, -25]})
+    .setLngLat(monument)
+    .setPopup(popup) // sets a popup on this marker
+    .addTo(map);
+
+</script>
```
Comment (kasunvp) [inline docs/_posts/examples/3400-01-27-set-popup.html]: just noticed while checking this out that it needs a newline here :)

**Pull request merged**